simbev/car.py

```diff
@@ -894,11 +906,11 @@ def _get_charging_usecase(self, power, extra_urban):
         elif self.home_parking and self.status == "home" and not self.home_detached:
             return "home_apartment"
         # TODO: decide on status an requirement for hpc
-        elif self.status == "hpc" and extra_urban:
+        elif (self.status == "hpc" and extra_urban) or charging_use_case == "highway_fast":
```

Comment:
We could also just return charging_use_case if its not None instead of checking it for every use case separately. Probably should add None as default value for charging_use_case